apps/dialtone-documentation/docs/.vuepress/baseComponents/BaseIcon.vue

```diff
@@ -54,6 +53,8 @@ import SpotMaleLaptopTyping from '@dialpad/dialtone/lib/dist/vue/spot/SpotMaleLa
 import SpotFileUpload from '@dialpad/dialtone/lib/dist/vue/spot/SpotFileUpload.vue';
 import SpotBrowserTableGraph from '@dialpad/dialtone/lib/dist/vue/spot/SpotBrowserTableGraph.vue';
 import SpotBrowserListCallout from '@dialpad/dialtone/lib/dist/vue/spot/SpotBrowserListCallout.vue';
+import SpotMind from '@dialpad/dialtone/lib/dist/vue/spot/SpotMind.vue';
+import SpotEmpty from '@dialpad/dialtone/lib/dist/vue/spot/SpotEmpty.vue';
```

Comment:
`EmptyState` is probably far too specific. Spot Illustration are mostly _illustative_ and thus shouldn't imply too much how they'd be used — even if this one might be used for an empty state.